Tasks/AzureAppServiceSettingsV1/task.json

```diff
@@ -0,0 +1,325 @@
+{
+    "id": "3eeea460-bffa-11e9-9cb5-2a2ae2dbcce4",
+    "name": "AzureAppServiceSettings",
+    "friendlyName": "Azure App Service Settings",
+    "description": "Update/Add App settings an Azure Web App for Linux or Windows",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks",
+    "helpMarkDown": "[Learn more about this task](https://aka.ms/azurewebappdeployreadme)",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "minimumAgentVersion": "2.104.1",
+    "groups": [
+        {
+            "name": "ApplicationAndConfigurationSettings",
+            "displayName": "Application and Configuration Settings",
+            "isExpanded": true
+        }
+    ],
+    "inputs": [
+        {
+            "name": "ConnectedServiceName",
+            "aliases": [
+                "azureSubscription"
+            ],
+            "type": "connectedService:AzureRM",
+            "label": "Azure subscription",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Select the Azure Resource Manager subscription"
+        },
+        {
+            "name": "WebAppName",
+            "type": "pickList",
+            "label": "App Service name",
+            "defaultValue": "",
+            "required": true,
+            "properties": {
+                "EditableOptions": "True"
+            },
+            "helpMarkDown": "Enter or Select the name of an existing Azure App Service"
+        },
+        {
+            "name": "ResourceGroupName",
```

Comment:
why is this always required

Reply:
We have made resource group intake as mandatory in the designer because we need it to display slot names. However, while taking input its not a mandatory input since we can extract resource group from web app name (for YAML). Already tested.